hedera-node/src/main/java/com/hedera/services/store/tokens/HederaTokenStore.java

```diff
@@ -148,10 +155,32 @@ public HederaTokenStore(
 		this.backingTokens = backingTokens;
 		this.tokenRelsLedger = tokenRelsLedger;
 		this.sideEffectsTracker = sideEffectsTracker;
-		this.uniqTokenViewsManager = uniqTokenViewsManager;
+		this.uniqueTokenViewsManager = uniqueTokenViewsManager;
 		/* Known-treasuries view is re-built on restart or reconnect */
 	}
 
+	@Override
+	protected ResponseCodeEnum checkAccountUsability(AccountID aId) {
```

Comment:
Where is this method used?

Reply:
This was overridden in order to exclude the HederaLedger from the Precompile logic, because we agreed that we are not going to be instantiating any HederaLedger in the PrecompileHTSContract file.